docs/presentation/Hierarchies/ChildNodeRule.md

```diff
@@ -1,34 +1,115 @@
-# ChildNodeRule
+# Child Node Rule
 
 > Based on [ChildNodeRule]($presentation-common) interface.
 
-Child node rules are used to define child nodes. Generally, `priority`, `condition`, `onlyIfNotHandled` and `stopFurtherProcessing` attributes are used to determine where (under
-which parent node) the branch is placed.
+Child node rules are used to define child nodes in a hierarchy.
+
+The rules have two types of attributes - for defining *placement* and for defining *branch content*. The latter attributes only get used if the rule does get used after evaluating the former.
 
```

Comment:
Consider using an em dash (—) if you want to sound informal, yet professional, or simply a colon (:) instead of a hyphen.

Reply:
changed to a colon